src/main/java/bit/javaoop/Employee.java

```diff
@@ -3,12 +3,13 @@
 public class Employee extends AbstractEmployee {
 
 
-    public Employee(String name, double salary) {
+    Employee(String name, double salary) {
         super(name, salary);
     }
 
     @Override
     public boolean isSatisfied() {
         return getSalary() > 10_000;
```

Comment:
Takie magiczne liczby jak to 10_000 można wyekstrahować do stałej i nazwać (skrót ctrl + alt + C ) np. SATISFYING_SALARY